openssl-dynamic/src/main/c/ssl_private.h

```diff
@@ -213,6 +217,11 @@ struct tcn_ssl_ctxt_t {
     apr_uint32_t            ticket_keys_renew;
     // The client presented a ticket that did not match any key in the list.
     apr_uint32_t            ticket_keys_fail;
+
+    jmethodID               verifier_method;
+    jmethodID               cert_requested_callback_method;
```

Comment:
@normanmaurer - Yes this was the intention. To pack memory tighter and avoid padding ... and not rely upon compiler optimizations/pragma definitions to do so. At one point in development I actually modified this structure but then realized I didn't need to ... I can factor this change out into another PR if you prefer.